docs/project.md

```diff
@@ -3,3 +3,12 @@ sidebar_position: 3
 ---
 
 # Project
+
+# Rules
+1. It is not allowed to use processors (eg Broadcom BCM2711, Broadcom BCM2712), you can only use microprocessors. Examples of microprocessors: RP2040, ATmega328, nRF52, PIC.
```

Comment:
```suggestion
1. It is not allowed to use processors (eg Broadcom BCM2711, Broadcom BCM2712), you can only use microprocessors. Examples of microprocessors: RP2040, nRF52, STM32 or ESP32 (RISC-V)
```

We cannot use Arduino or PIC, as there is not Rust support.

I would suggest rephrasing the rules and telling students what they *can* do, instead of what they *can't*.